app/views/to_change_everything/show.html.erb

```diff
@@ -33,7 +33,9 @@
     <h1><%= t '.take_flight_html' %></h1>
   </div>
 
-  <div class="spread" id="next">
-    <h1><%= link_to t('.next_text'), t('.next_path') %></h1>
-  </div>
+  <% if t('.next_text').present? %>
```

Comment:
this skips rendering `the secret is to begin` section if there isn't a translation for it